src/types/brand.ts

```diff
@@ -0,0 +1,4 @@
+export const enum Brand {
```

Comment:
I think we can keep this version, I missed that we typed it as an enum when adding the footer. It should work fine it think.